cmd/starter/main.go

```diff
@@ -76,98 +77,112 @@ func main() {
 	}, docker.AuthConfiguration{})
 	orFail(err)
 
-	redisClient.Incr("active-workers")
-	defer func() {
-		redisClient.Decr("active-workers")
-	}()
+	c := cron.New()
+	c.AddFunc("0 */5 * * * *", announceActiveWorker)
+	c.Start()
```

Comment:
is this removed when the process exits somehow? or is this added every time this runs?